docker/kubernetes-tentacle/scripts/configure-and-run.sh

```diff
@@ -230,39 +230,28 @@ function registerTentacle() {
     tentacle "${ARGS[@]}"
 }
 
-function run() {
-    echo "==============================================="
-    echo "Starting Octopus Deploy Kubernetes Tentacle"
-    echo "==============================================="
+setupVariablesForRegistrationCheck
+getStatusOfRegistration
 
-    # the exec here means that the host bash process is replaced by the tentacle process
-    exec tentacle agent --instance Tentacle --noninteractive
-}
+if [ "$IS_REGISTERED" == "true" ]; then
+    echo "Tentacle is already configured and registered with server."
+else
+    echo "==============================================="
+    echo "Configuring Octopus Deploy Kubernetes Tentacle"
 
+    validateVariables
```

Comment:
is there a reason this call is interleaved with the `echo` statements?

Reply:
I believe it's a formatting thing. It outputs the values of the variables in this block